web_responsive/static/src/components/apps_menu/apps_menu.esm.js

```diff
@@ -0,0 +1,335 @@
+/** @odoo-module **/
+/* Copyright 2018 Tecnativa - Jairo Llopis
+ * Copyright 2021 ITerra - Sergey Shebanin
+ * Copyright 2023 Onestein - Anjeel Haria
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+import {NavBar} from "@web/webclient/navbar/navbar";
+import {useAutofocus, useBus, useService} from "@web/core/utils/hooks";
+import {useHotkey} from "@web/core/hotkeys/hotkey_hook";
+import {scrollTo} from "@web/core/utils/scrolling";
+import {debounce} from "@web/core/utils/timing";
+import {fuzzyLookup} from "@web/core/utils/search";
+import {WebClient} from "@web/webclient/webclient";
+import {patch} from "web.utils";
+import {escapeRegExp} from "@web/core/utils/strings";
+
+const {Component, useState, onPatched, onWillPatch} = owl;
+
+// Patch WebClient to show AppsMenu instead of default app
+patch(WebClient.prototype, "web_responsive.DefaultAppsMenu", {
+    setup() {
+        this._super();
+        useBus(this.env.bus, "APPS_MENU:STATE_CHANGED", ({detail: state}) => {
+            document.body.classList.toggle("o_apps_menu_opened", state);
+        });
+    },
+});
+
+/**
+ * @extends Dropdown
+ */
+export class AppsMenu extends Component {
+    setup() {
+        super.setup();
+        this.state = useState({open: false});
+        this.menuService = useService("menu");
+        useBus(this.env.bus, "ACTION_MANAGER:UI-UPDATED", () => {
+            this.setOpenState(false, false);
+        });
+        this._setupKeyNavigation();
+    }
+    setOpenState(open_state, from_home_menu_click) {
+        this.state.open = open_state;
+        // Load home page with proper systray when opening it from website
+        if (from_home_menu_click) {
+            var currentapp = this.menuService.getCurrentApp();
+            if (currentapp && currentapp.name == "Website") {
+                if (window.location.pathname != "/web") {
+                    const icon = $(
+                        document.querySelector(".o_navbar_apps_menu button > i")
+                    );
+                    icon.removeClass("fa fa-th-large").append(
+                        $("<span/>", {class: "fa fa-spin fa-spinner"})
+                    );
+                }
+                window.location.href = "/web#home";
+            } else {
+                this.env.bus.trigger("APPS_MENU:STATE_CHANGED", open_state);
+            }
+        } else {
+            this.env.bus.trigger("APPS_MENU:STATE_CHANGED", open_state);
+        }
```

Comment:
I don't know the purpose of these code. I simply commented out and it solves my needs, that is keep stay on apps menu for me to select wanted apps when opening home from website.